api/v1beta1/openstackprovisionserver_webhook.go

```diff
@@ -99,14 +99,16 @@ var _ webhook.Defaulter = &OpenStackProvisionServer{}
 func (r *OpenStackProvisionServer) Default() {
 	openstackprovisionserverlog.Info("default", "name", r.Name)
 
-	if r.Spec.DownloaderImageURL == "" {
-		r.Spec.DownloaderImageURL = openstackProvisionServerDefaults.DownloaderImageURL
+	if r.Spec.OSContainerImageURL == "" {
+		r.Spec.OSContainerImageURL = openstackProvisionServerDefaults.OSContainerImageURL
 	}
 	if r.Spec.AgentImageURL == "" {
 		r.Spec.AgentImageURL = openstackProvisionServerDefaults.AgentImageURL
 	}
 	if r.Spec.ApacheImageURL == "" {
 		r.Spec.ApacheImageURL = openstackProvisionServerDefaults.ApacheImageURL
 	}
-
+	if r.Spec.OSImage == "" {
+		r.Spec.OSImage = "edpm-hardened-uefi.qcow2"
+	}
```

Comment:
Since this is included in the defaulting logic for the webhook, should there also be an `OSImage` in `OpenStackProvisionServerDefaults` and a corresponding ENV var default for it (rather than using the hardcoded value)?

Reply:
Yeah, no worries for the moment.  We can address it however we think is necessary in a later PR.

Reply:
Ack, I've the patch ready, would push it once this gets merged.